modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-rhv.adoc
+
+[id="machineset-yaml-rhv_{context}"]
+=  Sample YAML for a machine set custom resource on {rh-virtualization}
+
+This sample YAML defines a machine set that runs on {rh-virtualization} and creates nodes that are labeled with `node-role.kubernetes.io/<node_role>: ""`.
```

Comment:
@Gal-Zaidman Is this correct for RHV? For OpenStack it says:

This sample YAML defines a machine set that runs on Red Hat OpenStack Platform (RHOSP) and creates nodes that are labeled with node-role.openshift.io/<node_role>: ""

Notice: node-role.openshift.io instead of what we have now: node-role.kubernetes.io

Reply:
Yes for us it is:
node-role.kubernetes.io/<node_role>: ""